packages/devextreme/js/renovation/ui/scheduler/workspaces/base/header_panel/date_header/layout.tsx

```diff
@@ -73,7 +73,7 @@ export const viewFunction = ({
               dateCellTemplate={dateCellTemplate}
               key={key}
               colSpan={colSpan}
-              splitText={isMaterial}
+              splitText={isMaterial || isFluent}
```

Comment:
I have already seen several uses 'isMaterial || isFluent', maybe you can make a common uses line 'isMaterialBased'?